README.md

```diff
@@ -312,6 +312,38 @@ The following sample plugins exist for NRI:
 Please see the documentation of these plugins for further details
 about what and how each of these plugins can be used for.
 
+## Security Considerations
+
+From a security perspective NRI plugins should be considered part of the
+container runtime. NRI does not implement granular access control to the
+functionality it offers. Access to NRI is controlled by restricting access
+to the systemwide NRI socket. If a process can connect to the NRI socket
+and send data, it has access to the full scope of functionality available
+via NRI.
+
+The same precautions and principles apply to protecting the NRI socket as
+to protecting the socket of the runtime itself. Unless it already exists,
+NRI itself creates the directory to hold its socket with permissions that
+allow access only for the user ID of the runtime process. In practice this
+limits NRI access to processes running as root (UID 0).
```

Comment:
How do you feel about changing "In practice" to "By default"? I think we'd want to recommend here that users don't change the default ownership or permissions of the directory or socket file.

Reply:
Thanks ! I'll update the proposed additions about security consideration with this (and the above comments).